app/policies/application_policy.rb

```diff
@@ -0,0 +1,53 @@
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
```

Comment:
Should we define all the controller actions upfront? Not every controller will have all 7 of these actions. I'd prefer we define only the ones we need in the policy subclasses, but I can be convinced otherwise.

Reply:
I lean slightly toward doing it this way because it's the default for pundit, but much like you, I can be swayed. I think I know why you'd rather define the necessary ones in policy subclasses, but just in case I'm wrong, could you elaborate?

Reply:
Not every controller will have all 7 of these actions, so some of these will be unused. Even in controllers with most of these actions, these will not necessarily be useful defaults.

For example: AdminPostsController has an index action, but `AdminPostPolicy#index?` is not used, because we don't lock down anything there. For the 5 actions that do use `authorize`, we end up overriding the ApplicationPolicy methods in AdminPostPolicy anyway.

That said, I'm also ok with having superclass defaults. The defaults block access to everything, so it's not a big deal if we miss overriding one of them.

Reply:
So just to be clear, are you asking for any changes here?

Reply:
I'm checking right now to see if anyone else on the committee has strong feelings about this and I'll let you know where we end up.

Reply:
I think we'll leave this as it is. Thanks!